modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc

```diff
@@ -1,4 +1,7 @@
-= Database Configuration
+= Database Configuration on Linux
+:toc: right
+
+== Introcuction
```

Comment:
s/Introcuction/Introduction/g

Reply:
I have created PR https://github.com/owncloud/docs/pull/957 to fix this, as this must be corrected in master and 10.0/10.1